src/test/feature/lib/file_replace.h

```diff
@@ -0,0 +1,26 @@
+#ifndef SRC_TEST_FEATURE_LIB_FILE_REPLACE_H_
+#define SRC_TEST_FEATURE_LIB_FILE_REPLACE_H_
+
+#include <string>
+#include <map>
+
+namespace hawq {
+namespace test {
+
+class FileReplace
+{
+public:
+	FileReplace() {}
+	~FileReplace() {}
```

Comment:
Do you mean not to define the copy constructor here? If you don't define anyone of constructor, de-constructor, copy constructor, the compiler will auto create them. If you do define some of them, the other could not be auto created.